### PR TITLE
Publish FAR to Stable channel instead of Candidate

### PR DESCRIPTION
**Pull request by razo7**
Getting ready to new release and publish to the `stable` channel.

### CODE REVIEW

**Comment by razo7** [comment]
/pj-rehearse auto-ack
/pj-rehearse max

/hold
Waiting until other active PRs will be merged.

**Review by slintes — APPROVED** [review]

**Pull request merged**
